backend/src/domains/base.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { PrimaryGeneratedColumn, Column, UpdateDateColumn, CreateDateColumn } from 'typeorm';
+
+export abstract class BaseEntity {
+    // @PrimaryGeneratedColumn()
+    // @Column({ type: 'numeric' })
+    // public id: number;
+
+    @CreateDateColumn({ name: 'created_at', type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
+    createdAt: Date;
+
+    @Column({ name: 'created_by', type: 'numeric' })
+    createdBy: number;
```

Comment:
Probably createdBy and updatedBy also not a part of a very basic generic entity. Not all tables in our database will have a user that will fill these tables. I guess that will be a lot of tables, that will be just directories. Probably we should split this basic generic model on two - BaseEntity and for example - UserManagebleEntity?

Reply:
I assumed that only the date and the author of the changes will be stored in the base entity. I see no reason to store fields "created_at" and "updated_at" at the table without binding it to the user. What will this information give us? If we don’t care which user made changes to the table, then we don’t have to inherit this entity from the base entity.

Reply:
The case with the necessity of created_at and without the necessity of created_by is when something is filling in some dictionaries by portions.